examples/near_global_ocean_simulation.jl

```diff
@@ -212,6 +225,9 @@ sn = @lift begin
     parent(un) .= parent(u[$n])
     parent(vn) .= parent(v[$n])
     compute!(s)
+    sn = interior(s)
+    sn[land] .= NaN
+    view(sn, :, :, 1)
```

Comment:
Why don't we use the Makie extension here?

Reply:
Makie extension doesn't work for sliced fields (ones that correspond e.g. to `indices = (:, :, 40:40)`); I was about to make a MWE.

Similar for T and e...

Reply:
ok would be good to have an issue

Reply:
is this closed with recent PR in oceananignas?

Reply:
https://github.com/CliMA/Oceananigans.jl/issues/3921